src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,22 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final String LOGGER_DEBUG_MESSAGE = "Authentication error has occurred "
+                                                        + "while authentication with login=";
+    private static final Logger logger = LogManager.getLogger("mate.academy");
+
     public static void main(String[] args) {
+
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.debug(LOGGER_DEBUG_MESSAGE);
```

Comment:
I guess it's better to use the ERROR log level here.

Reply:
pushed wrong commit =)